pkg/workloads/spark_job/test/integration/iris_test.py

```diff
@@ -17,7 +17,7 @@
 from spark_job import spark_job
 from lib.exceptions import UserException
 from lib import Context
-from test.integration.iris_context import raw_ctx
+from test.integration.iris_context import get_raw_ctx
```

Comment:
Maybe we should cal the method `get()`, and then call `iris_context.get()`? Just a thought, could go either way.